aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,336 @@
+
+.. image:: localization/_static/newi.png
+
+2. In the **Search installed templates** box, enter "resource" and name the file.
+
+.. image:: localization/_static/res.png
+
+3. Enter the key value (native string) in the **Name** column and the translated string in the **Value** column.
+
+.. image:: localization/_static/hola.png
+
+Visual Studio shows the *Welcome.es.resx* file.
+
+.. image:: localization/_static/se.png
+
+Later on in this article I'll show how your ASP.NET app can use the key "Hello" to fetch the string "Hola".
+
+Generating resource files with Visual Studio
+-----------------------------------------------
+
+If you create a resource file in Visual Studio without a culture in the file name (for example, *Welcome.resx*), Visual Studio will create a C# class with a property for each string.  That's usually not what you want with ASP.NET Core; you typically don't have a default *.resx* resource file (A *.resx* file without the culture name). We suggest you create the *.resx* file with a culture name (for example *Welcome.fr.resx*). When you create a *.resx* file with a culture name, Visual Studio will not generate the class file.
+
+If you remove the ".fr" culture designator AND you have the culture set to French (via cookie or other mechanism), the default resource file is read and strings are localized. This is expected behavior. \*.resx designates a default or fallback resource, so that when nothing meets your requested culture you're served \*.resx. If you want to just return the key when missing a resource for the requested culture you must not have a default resource file.
+
+
+Adding Other Cultures  
+-----------------------
+ 
+Each language and culture combination (other than the default language) requires a unique resource file. You can create resource files for different cultures and locales by creating new resource files in which the ISO language codes are part of the file name (for example, **en-us**, **fr-ca**, and **en-gb**). These ISO codes are placed between the file name and the .resx file name extension, as in *Welcome.es-MX.resx* (Spanish/Mexico). To specify a culturally neutral language, you would eliminate the country code, such as Welcome.fr.resx for the French language. 
+
+
+Preparing your app for localization
+----------------------------------------
+
+Introduced in ASP.NET Core, ``IStringLocalizer`` was architected to improve productivity when developing localized apps. The simple interface has an indexer and an ``IEnumerable`` for returning localized strings. ``IStringLocalizer`` doesn't require you to store the default language strings in a resource file. You can develop an app targeted for localization and not need to create resource files early in development. We anticipate that many developers will **not** create a default language resource file. The code below shows how to wrap the string "About Title" for localization.
```

Comment:
`We anticipate that many developers will **not** create a default language resource file.` is a bit out of place here, no?